spec/requests/05_patch_route_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+RSpec.describe "PATCH /ratingQuestions/:id" do
+  context "when the question exists" do
+    let!(:question) do
+      RatingQuestion.create!(title: "Hello World!", tag: "World")
+    end
```

Comment:
Put a space here between the `let` and the `it`.

Reply:
ok